### ISSUE TITLE
A user can receive as many invites to the same community as the invite button is pressed multiple times

### ISSUE THREAD

**Issue by abhi-status** [status-im/status-desktop#5303]
# Bug Report

NOTE: this wouldn't be fixed for now, but we should close the modal when the invite is done

## Description

When a user hits `Invite` multiple times to invite a contact to a community, the recipient sees multiple invites to the same community.

## Steps to reproduce

- Create a Community
- Invite a contact to join this community
- Press invite again and again

#### Expected behavior
The recipient must only see this invite sent in once.

#### Actual behavior
![image](https://user-images.githubusercontent.com/95302135/161305440-e9369fc5-d75b-4e77-b010-c54b51312719.png)

### Additional Information

- Status desktop version: Master :e26befb6c26141529a5c4263bc8b26aa7c27e042

**Comment by iurimatias** [comment]
the modal itself should close

**Comment by PascalPrecht** [comment]
This issue is not a bug and somewhat obsolete. Here why:

1. The bubbles shown in the chat are currently "invites" sent to a contact. When people send 3 invites, the receiving end sees 3 invitation bubbles. This is by design. It is most likely unexpected that invites can only be sent once. So seeing multiple such bubbles is correct and not a bug.

2. We are moving away from the concept of invites. Users will not be invited anymore, instead, community owners (and other members) can **share** a community. This already exists today and there are 2 ways to do this:

a) Community owner triggers the `ShareCommunity` API which will result in the same "invite" bubble in the receiving user's chat. However there's no "invite" associated to that.

b) A share link of a community can be sent as normal chat message. The receiving user will then just see the link, which will automatically expanded to a bubble similar to a)

In **all** cases, users will have to **request access** to the community (even though the label on the bubble might say "join").
When a request to join was performed, the request will either be manually approved by the community owner or admin, or, if configured that way, the request will be automatically approved.

So to summarize:

1. Seeing multiple bubbles is by design. This is true for "sharing" communities as well. If I share it 3 times, you'll see 3 bubbles
2. The bubbles are most likely here to stay
3. Hitting "join" will actually result in a "request to join" request

Closing this as "won't fix", also as discussed with @alexandraB99 and @John-44
